web/src/repo/LSIFVerification.tsx

```diff
@@ -52,12 +39,65 @@ export class LSIFVerification extends React.PureComponent<Props, State> {
 
     public componentDidMount(): void {
         this.subscriptions.add(
-            from(fetchChallenge())
-                .pipe(catchError(error => [asError(error)]))
-                .subscribe(challengeOrError => {
-                    this.setState({ challengeOrError })
-                })
+            ajax({
+                url: new URL('/.api/lsif/challenge', window.location.href).href,
```

Comment:
```suggestion
                url: '/.api/lsif/challenge',
```
pretty sure a relative URL works